### DISCUSSION TITLE
Sync database Structures/Migrate should be implemented

### DISCUSSION THREAD

**Discussion by AtillaSE** [dbeaver/dbeaver#17620] [Ideas]
In the options there's already a Compare/Migrate function, but the Migrate is not implemented yet. There is a workaround by exporting a database and importing that again on the other server, but that will also copy over the data. I'd like to be able to 'deploy' changes to the database structure without also migrating the data.

**Comment by uslss** [comment]
@AtillaSE Schema migration is already implemented in DBeaver PRO versions, if I catch your request correctly.
You could read more about it here https://github.com/dbeaver/dbeaver/wiki/Schema-compare
